src/Halogen/HTML/Properties.purs

```diff
@@ -201,6 +203,9 @@ src = prop (PropName "src")
 style :: forall r i. String -> IProp (style :: String | r) i
 style = attr (AttrName "style")
 
+scope :: forall r i. ScopeValue -> IProp (scope :: ScopeValue | r) i
```

Comment:
```suggestion
scope :: forall r i. I.ScopeValue -> IProp (scope :: I.ScopeValue | r) i
```

If you accept the other suggestion, this will need doing too 🙂